### DISCUSSION TITLE
Ground Items: Hide/show stackable items based on quantity.

### DISCUSSION THREAD

**Discussion by Skrendy** [runelite/runelite#15005] [Q&A] [answered]
It would be very handy to be able to hide stackable ground items, based on the quantity.
Example: I want to hide death runes if the stack contains less than 10, but i want them to be visible from 10 and above.

This existing pull request from 2019 seems to cover the feature:
https://github.com/runelite/runelite/pull/8146

**Comment by Enriath** [comment, marked as answer]
This is already possible. Add `death rune<10` to the hidden list. The same idea can be applied to highlighting above a certain amount.

**Reply by Skrendy** [reply]
Thank you! Exactly what i was looking for.
